src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -26,6 +32,11 @@
 
     private final Person toAdd;
 
+    @Override
+	public boolean isMutating(){
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)